api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/jena/RDFUploadController.java

```diff
@@ -315,7 +317,7 @@ private long operateOnModel(WebappDaoFactory webappDaoFactory,
                                 OntModelSelector ontModelSelector,
                                 boolean remove,
                                 boolean makeClassgroups,
-                                String userURI) {
+                                String userURI, String userId) {
```

Comment:
one parameter per line (it might be fixed by code style formatting, but please fix this if it is possible now :-) )

Reply:
fixed